llama_index/llms/openai_like.py

```diff
@@ -22,34 +23,46 @@ class OpenAILike(OpenAI):
 
     context_window: int = Field(
         default=DEFAULT_CONTEXT_WINDOW,
-        description="The maximum number of context tokens for the model.",
+        description=LLMMetadata.__fields__["context_window"].field_info.description,
+    )
+    num_output: Optional[int] = Field(
```

Comment:
why do we need `num_output` as well as `max_tokens`? Don't they represent the same thing?

Reply:
Haha nice catch, reading the added code here, it does seem like I added some indirection 😅 .

I will investigate this later when making the demo notebook, and likely conclude to remove this extra `Field` 👍